### PR TITLE
Add context and tracing to attestation pool

### PR DESCRIPTION
**Pull request by terencechain**
This PR adds context and tracing to attestation pool

### CODE REVIEW

File: beacon-chain/core/blocks/block_operations.go [hunk 1]
```diff
@@ -445,10 +447,10 @@ func VerifyAttesterSlashing(beaconState *pb.BeaconState, slashing *ethpb.Atteste
 	if !IsSlashableAttestationData(data1, data2) {
 		return errors.New("attestations are not slashable")
 	}
-	if err := VerifyIndexedAttestation(beaconState, att1); err != nil {
+	if err := VerifyIndexedAttestation(context.TODO(), beaconState, att1); err != nil {
```
Comment (terencechain) [inline beacon-chain/core/blocks/block_operations.go]: Is `Todo()` ok here? Since we don't care about tracing for `VerifyAttesterSlashing`

**Comment by 0xKiwi** [comment]
Have you checked what a trace of this looks like?

File: beacon-chain/core/blocks/block_operations.go (hunk 1 above)
Comment (terencechain) [reply beacon-chain/core/blocks/block_operations.go]: Talked with @prestonvanloon, will follow best practice and pass a real ctx here

**Comment by terencechain** [comment]
> Have you checked what a trace of this looks like?

Yep. Can check it when we apply it to the cluster

**Pull request merged**